Automattic/newspack-listings
# Update CPT icon and Block icons

### All Submissions:






### Changes proposed in this Pull Request:

* Add `@wordpress/icons` component and update `@automattic/calypso-build`
* Create new custom icon for the CPT/Block
* Use `@wordpress/icons` for inner blocks

![1](https://user-images.githubusercontent.com/177929/112994641-5d1df900-9162-11eb-8c91-88917bf4db5d.png)
![2](https://user-images.githubusercontent.com/177929/112994644-5db68f80-9162-11eb-8286-21237072ec2b.png)
![3](https://user-images.githubusercontent.com/177929/112994646-5e4f2600-9162-11eb-85de-ce051fa550b1.png)

### How to test the changes in this Pull Request:

1. Switch to this branch
2. In wp-admin, do you see the new menu icon?
3. In the editor, do the placeholder have the new icons?
4. In the main block inserter, do you see the new icon in blue?
5. In the child block inserter, do you see various new icons depending on the listing type?

### Other information:
